Makefile

```diff
@@ -0,0 +1,45 @@
+COMPOSE_DEV		:= docker-compose
+COMPOSE_TEST		:= docker-compose -f docker-compose.test.yml
+BASH_DEV		:= $(COMPOSE_DEV) run vets-api bash --login -c
+BASH_TEST		:= $(COMPOSE_TEST) run vets-api bash --login -c
+
+.PHONY: default
+default: ci
+
+.PHONY: ci
+ci:
+	@$(BASH_TEST) "bundle exec rake db:setup db:migrate ci"
+
+.PHONY: console
```

Comment:
I'm not convinced `console` should depend on `db` - at least in my typical workflow when I want to use rails console I don't want to run migrations.